src/App.css

```diff
@@ -1,3 +1,64 @@
 .App {
-  text-align: center;
+  width: 100%;
```

Comment:
Correct me if I'm wrong, the default width for divs in 100%.
There is no need for this line

Reply:
I thought it might be a 'better safe than sorry' situation, where maybe there is some weird browser on which it is not the case, isn't it worth staying there?

Reply:
As far as I know, this is pretty basic behaviour of divs, I think that if Chrome and IE supports it then you are safe enough.

Reply:
Implemented.